packages/dd-trace/test/config.spec.js

```diff
@@ -513,6 +528,8 @@ describe('Config', () => {
     process.env.DD_APPSEC_WAF_TIMEOUT = 11
     process.env.DD_APPSEC_OBFUSCATION_PARAMETER_KEY_REGEXP = '^$'
     process.env.DD_APPSEC_OBFUSCATION_PARAMETER_VALUE_REGEXP = '^$'
+    process.env.DD_APPSEC_HTTP_BLOCKED_TEMPLATE_HTML = BLOCKED_TEMPLATE_HTML_PATH
+    process.env.DD_APPSEC_HTTP_BLOCKED_TEMPLATE_JSON = BLOCKED_TEMPLATE_JSON_PATH
```

Comment:
the test is called `should give priority to the options` so if you set the same value in both env vars and programmatic options then it defeats the point of the test, you need to have different values in env vars and programmatic config, and assert that the programmatic config has priority

Reply:
true

Reply:
Actually you're not testing priority now, because if the env var fail to read it will return undefined, so you're only testing the log message here not the priority

Reply:
i'll fix it

Reply:
https://github.com/DataDog/dd-trace-js/pull/2913/commits/110f643f5e37ed549a486fc5832e3f0802c7fc8a